tests/saved_mapped.nf.test.snap

```diff
@@ -33,16 +33,19 @@
                 "reference"
             ],
             [
-
+                
```

Comment:
Trailing whitespace?

Reply:
that's generated by nf-test, It's because I'm snapshoting an empty list.
I'll see about that in subsequent PRs.
Thanks @erikrikarddaniel